pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -130,6 +130,9 @@ int kprobe__tcp_close(struct pt_regs* ctx) {
     u64 pid_tgid = bpf_get_current_pid_tgid();
     sk = (struct sock*)PT_REGS_PARM1(ctx);
 
+    // Should actually delete something only if the connection never got established
+    bpf_map_delete_elem(&tcp_ongoing_connect_pid, &sk);
```

Comment:
Are we going to do something with this information, i.e. that the connection failed? We could add some telemetry here for a start. Maybe for a future PR.

Reply:
Yes, that is what I plan to do to detect and count client-side failing TCP connections.